src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Colors.values().length);
+        String randomColor;
+        randomColor = "" + Colors.values()[randomIndex];
+        return randomColor;
```

Comment:
```suggestion
        int randomIndex = new Random().nextInt(Colors.values().length);
        String randomColor = Colors.values()[randomIndex].name();
        return randomColor;
```

мы можем использовать метод `.name()` для получения String из enum

Reply:
Что касательно этого комментария?

Reply:
як язрозумів метод name() можна використовувати якщо ми знаємо ім'я enum, щось на зразок Colors.RED.name(),
і перепробувавши всі варіанти про які я подумав, я не зміг використати метод name()

Reply:
Colors.values()[randomIndex] вернет нам конкретный экземпляр списка Colors, и мы можем на нем вызвать .name(). Ты пытался запустить и продебажить предложенный мною код?